tests/integration/state.go

```diff
@@ -114,6 +113,30 @@ func newEventState(bus *event.Bus) (*eventState, error) {
 	}, nil
 }
 
+// nodeState contains all testing state for a node.
+type nodeState struct {
+	// The node active in this test.
+	node *node.Node
+	// The node's client active in this test.
+	client clients.Client
```

Comment:
> suggestion: node.Node and clients.Client seem to basically be the same thing, it looks like it would be minimal hassle to get rid of clients.Client and just call nodeState.node.Peer.Connect(...).

You mean to get rid of `nodeState`'s `client` and instead of for example:
```go
node.client.GetCollections(s.ctx, client.CollectionFetchOptions{})
```
call
```go
node.DB.GetCollections(s.ctx, client.CollectionFetchOptions{})
```
(assuming your first suggestion is applied).

Reply:
Ah - my bad, when I made that suggestion I though `node.Node` implemented `client.DB` and you could have done `node.GetCollections`.  Don't bother with that suggestion :)